### DISCUSSION TITLE
Student verification rejection

### DISCUSSION THREAD

**Discussion by saimaheshtaduri** [community/community#62643] [GitHub Education]
### Hi everyone,

Hello, I recently applied for student verification but I got rejected saying sufficient information isn’t available. I have attached my college ID. I wanted to know what else am I missing. 
Any help is appreciated. 

Thanks !

**Comment by HvTung04** [comment]
Is your school ID in english?

**Comment by prabhjotschugh** [comment]
You can try to upload your school's ID card (with validity mentioned on it) or your recent fee receipt as a proof that you are still a student. Make sure the validity is mentioned in your ID card and clearly visible.
Hope it helps!
